lms/djangoapps/course_api/views.py

```diff
@@ -52,6 +52,7 @@ class CourseDetailView(DeveloperErrorViewMixin, RetrieveAPIView):
             * `"string"`: manually set
             * `"timestamp"`: generated form `start` timestamp
             * `"empty"`: the start date should not be shown
+        * pacing: Course pacing. Possible values: instructor, self
```

Comment:
In documentation we refer to "self-paced" and "instructor-paced" courses, so those values are fine.